OpenRA.Game/WorldUtils.cs

```diff
@@ -44,8 +44,8 @@ public static IEnumerable<Actor> FindActorsInCircle(this World world, WPos origi
 
 		public static bool HasVoices(this Actor a)
```

Comment:
I was going to object, but I see these three methods have a total of 2 usages, so I'm, inclined to let it go.